README.md

```diff
@@ -150,4 +151,26 @@ The sdk.js contains a function to automatically adjust the component height to t
 }
 ```
 
+## Incorporating changes from the boilerplate
```

Comment:
These instructions are for ourself?

Reply:
Although externals could do the same (and send a PR), I expect that it will be us to do this mostly. Better ideas where to put it?